### PR TITLE
Yelpapi

### PR DESCRIPTION
**Pull request by holmesm8**
@caachz PR for what we have worked on so far. Testing, environment variables.

### CODE REVIEW

**Comment by holmesm8** [comment]
@caachz Updated PR. We're getting the JSON objects we want to pretty excited about that. Take a look over, there's definitely opportunities to refactor but right now it does it's job.

File: app/controllers/restaurants_controller.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+
+# should recieve latitude, longitude, radius, price parameters
+class Server < Sinatra::Base
+  before do
+    content_type :json
+  end
+
+  get '/search' do
+    yelp = YelpService.new(params["latitude"], params["longitude"], params["radius"], params["price"])
```
Comment (agallant121) [inline app/controllers/restaurants_controller.rb:11]: Nice job averaging the reviews from two sources.

File: app/poros/restaurant.rb [hunk 2]
```diff
@@ -0,0 +1,32 @@
+class Restaurant
```
Comment (agallant121) [inline app/poros/restaurant.rb:1]: This poro seems very solid to me.

File: app/poros/restaurant.rb [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+              :location,
+              :categories
+  attr_accessor :google_rating,
+                :avg_rating
+
+  def initialize(restaurant)
+    @id = restaurant[:id]
+    @name = restaurant[:name]
+    @yelp_rating = restaurant[:rating].to_s
+    @google_rating = nil
```
Comment (agallant121) [inline app/poros/restaurant.rb:18]: Do these each start as nil by default?

File: app/services/yelp_services.rb [hunk 4]
```diff
@@ -0,0 +1,17 @@
+class YelpService
+  attr_reader :restaurant
+
+  def initialize(latitude, longitude, radius, price)
+    conn = Faraday.new("https://api.yelp.com/v3/businesses/search") do |f|
```
Comment (agallant121) [inline app/services/yelp_services.rb:5]: Could this Faraday.new be pulled out into private? Probably unnecessary for our app at this point but maybe for future iterations?

Comment (agallant121) [reply app/services/yelp_services.rb:5]: I might have read this wrong too and it is already broken up? @holmesm8 @caachz

File: app/controllers/restaurants_controller.rb (hunk 1 above)
Comment (TravisBorgsmiller) [reply app/controllers/restaurants_controller.rb:11]: Sinatra has some interesting controller methods

**Pull request merged**